JenkinsSolutions/q3.txt

```diff
@@ -15,3 +15,7 @@ pipeline {
         }
     }
 }
+
+//cron expression
+//H H * * 1,2,3,4,5
+
```

Comment:
correct! but can you think of an easier way of doing this? since we go from 1 through 5